x-pack/plugins/security_solution/common/endpoint/service/authz/authz.test.ts

```diff
@@ -182,6 +187,16 @@ describe('Endpoint Authz service', () => {
         ['canReadBlocklist', ['writeBlocklist', 'readBlocklist']],
         ['canWriteEventFilters', ['writeEventFilters']],
         ['canReadEventFilters', ['writeEventFilters', 'readEventFilters']],
+        // all dependent privileges are false and so it should be false
+        [
+          'canAccessResponseConsole',
+          [
```

Comment:
can you get this list using the `const` you defined so that its not hard coded?

Reply:
a40406782fb84e102b0470db7912ba893e3655ec